src/main/java/com/mycompany/app/App.java

```diff
@@ -13,12 +13,15 @@ public static void main( String[] args )
         // Generate some findbugs perhaps
         String a = "hello";
 		String b = "goodbye";
+		String c = "asdf";
 
-		if( a == b )
+		if( a == c)
 		{
 			System.out.println( "Hello World!" );
 		}
 
+
+
 		String aString = "bob";
 		  b.replace('b', 'p');
```

Comment:
**Reporter**: Findbugs
**Rule**: RV_RETURN_VALUE_IGNORED
**Severity**: ERROR
**File**: src/main/java/com/mycompany/app/App.java L26
**Source**: com.mycompany.app.App

Method ignores return value


   <p> The return value of this method should be checked. One common
cause of this warning is to invoke a method on an immutable object,
thinking that it updates the object. For example, in the following code
fragment,</p>
<blockquote>
<pre>
String dateString = getHeaderField(name);
dateString.trim();
</pre>
</blockquote>
<p>the programmer seems to be thinking that the trim() method will update
the String referenced by dateString. But since Strings are immutable, the trim()
function returns a new String value, which is being ignored here. The code
should be corrected to: </p>
<blockquote>
<pre>
String dateString = getHeaderField(name);
dateString = dateString.trim();
</pre>
</blockquote>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<612344640>*